src/objective-c/GRPCClient/GRPCCall.h

```diff
@@ -155,6 +155,19 @@ typedef NS_ENUM(NSUInteger, GRPCErrorCode) {
 };
 
 /**
+ * Flags for options of a gRPC call
+ *
+ */
+typedef NS_OPTIONS(NSUInteger, GRPCCallFlags) {
+  /** Signal that the call is idempotent */
+  GRPCFlagIdempotentRequest = 0x00000010,
+  /** Signal that the call should not return UNAVAILABLE before it has started */
+  GRPCFlagIgnoreConnectivity = 0x00000020,
```

Comment:
I see, but why mobile apps do not want to set these flags? I kind of think it reasonable for a mobile device to make an RPC service call marked as Idempotent and uses QUIC for this call.